### PR TITLE
Web-Console: add actions icon to lookups

### PR DESCRIPTION
**Pull request by mcbrewster**
Screen Shot 2019-05-24 at 3 57 22 PM

The actions are now accessible via the wrench icon, which is constant with the tasks and supervisors views. the actions are obtained via the getlookupActions functions, although the same actions are returned for all types of lookups, this way of getting actions is constant with tasks actions and supervisors actions and can easily be filtered if necessary.

### CODE REVIEW

File: web-console/src/dialogs/table-action-dialog/table-action-dialog.scss [hunk 1]
```diff
@@ -36,6 +36,9 @@ $side-bar-width: 120px;
     }
   }
 
+  .bp3-button{
+    text-align: center;
```
Comment (vogievetsky) [inline web-console/src/dialogs/table-action-dialog/table-action-dialog.scss]: what is this?

File: web-console/src/views/lookups-view/lookups-view.tsx [hunk 2]
```diff
@@ -54,9 +56,11 @@ export interface LookupsViewState {
   lookupEditSpec: string;
   isEdit: boolean;
   allLookupTiers: string[];
+  lookupActionDialogId: string | null;
```
Comment (vogievetsky) [inline web-console/src/views/lookups-view/lookups-view.tsx]: clean this up, not needed

File: web-console/src/views/task-view/tasks-view.tsx [hunk 3]
```diff
@@ -648,6 +656,7 @@ ORDER BY "rank" DESC, "created_time" DESC`);
             width: 70,
             filterable: false,
             Cell: row => {
+              console.log(row);
```
Comment (vogievetsky) [inline web-console/src/views/task-view/tasks-view.tsx]: no log please

**Comment by clintropolis** [comment]
Should there be some sort of text or warning .. somewhere .. that not all lookups support introspection so this functionality might not work depending on the type of lookup?

**Comment by mcbrewster** [comment]
> Should there be some sort of text or warning .. somewhere .. that not all lookups support introspection so this functionality might not work depending on the type of lookup?

I ended up having to remove everything that had to do with introspection, apparently that part of the api is going in the future. All that is left is edit and kill are under the wrench icon now.

**Comment by clintropolis** [comment]
>I ended up having to remove everything that had to do with introspection, apparently that part of the api is being removed in the near future. All that is left is edit and kill are under the wrench icon now.

Oops, I did not refresh my browser before looking through this code and was seeing the old stuff, it looks a lot different now :+1:

File: web-console/src/views/task-view/tasks-view.tsx [hunk 4]
```diff
@@ -176,6 +176,7 @@ export class TasksView extends React.Component<TasksViewProps, TasksViewState> {
     this.supervisorQueryManager = new QueryManager({
       processQuery: async (query: string) => {
         const resp = await axios.get('/druid/indexer/v1/supervisor?full');
+        console.log(resp.data);
```
Comment (clintropolis) [inline web-console/src/views/task-view/tasks-view.tsx]: console.log

File: web-console/src/views/task-view/tasks-view.tsx [hunk 5]
```diff
@@ -219,12 +219,12 @@ export class TasksView extends React.Component<TasksViewProps, TasksViewState> {
     //   FAILED => 1
 
     this.taskQueryManager.runQuery(`SELECT
-  "task_id", "type", "datasource", "created_time",
+  "task_id", "type", "datasource", "created_time", "location",
```
Comment (vogievetsky) [inline web-console/src/views/task-view/tasks-view.tsx]: can you remove this change?

File: web-console/src/views/lookups-view/lookups-view.tsx [hunk 6]
```diff
@@ -326,33 +351,35 @@ export class LookupsView extends React.Component<LookupsViewProps, LookupsViewSt
...
+            columns={tableColumns}
+            onChange={(column) => tableColumnSelectionHandler.changeTableColumnSelection(column)}
+            tableColumnsHidden={tableColumnSelectionHandler.hiddenColumns}
+          />
+        </ViewControlBar>
+        {this.renderLookupsTable()}
+        {this.renderLookupEditDialog()}
+        {this.renderDeleteLookupAction()}
+      </div>;
+    </>;
```
Comment (vogievetsky) [inline web-console/src/views/lookups-view/lookups-view.tsx]: why is this wrapped in a fragment?

**Comment by vogievetsky** [comment]
looks good to me after travis 👍

**Pull request merged**